contracts/accounts/Account.sol

```diff
@@ -340,7 +340,9 @@ contract Account is
 
     mapping(address => bool) private _upgradedTo;
 
-    function migrateState(address oldImplementation) internal onlyInitializing {
+    function migrateState(
+        address // _oldImplementation
+    ) internal onlyInitializing {
```

Comment:
don't do this, it messes up compilation and export of contract interfaces to other languages